CMSgov/mint-app
# [EASI-2752] Fixing FE errors due to changing the way Operational Solution is added/updated

# EASI-2752

## Changes and Description

- Removed `UpdateCustomOperationalSolution.ts` `UpdateCustomOperationalSolutionByID.ts`, and `UpdateOperationalNeedSolution.ts` and its corresponding types
- Added `CreateOperationalSolution.ts` and `UpdateOperationalSolution.ts` and its corresponding types
- Edited `solutionmap.ts` since there were some OperationalSolutionKey that were removed
- Updated tests that were using an outdated OperationalSolutionKey




## How to test this change

1. Go through the whole process of adding a operational solution to any operational need 
3. Go through the whole process of adding a CUSTOM solution to any need
4. Go through the whole process of updating a solution
5. Ensure there are no errors
6. Try to break it


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
